### ISSUE TITLE
Add a sample ring handler to the example application

### ISSUE THREAD

**Issue by StasPerekrestov** [bhauman/lein-figwheel#66]
Hello,
I haven't been very successful in making my application's routings to work with the figwheel http-kit server.
So, maybe it'll be reasonable to add a simple handler (e.g. json response handler) to the [example application](https://github.com/bhauman/lein-figwheel/tree/master/example) to demonstrate
how figwheel behaves with the specified :ring-handler parameter.

Thanks!

**Comment by bhauman** [comment]
Hey just wanted to let you know I saw this.  I really need to redo the example application in general.  

Were you able to get your handler working?

**Comment by StasPerekrestov** [comment]
Hello.
I've found a handler in your example application and I was able to make the handler work.
However, I still can't make it work in my project. I suppose, that the main cause is misconfiguration.
And I have to check the configuration more carefully.

I'm developing a sample [application](https://github.com/StasPerekrestov/rubomg/) to learn clojure-related stuff.
In the branch websockets-sente, I [host](https://github.com/StasPerekrestov/rubomg/blob/websockets-sente/src/clj/rubomg/serve.clj) the application using http-kit. 
So, I've decided to play with figwheel (It's a really mind blowing thing ;) )
Because the application uses websockets to channel data from server I decided to embed
a handler into the figwheel http-kit server. Well, as I've already mentioned above it seems to me that the problem is in misconfiguration and I haven't just found out what is wrong.

The fun thing is that I copy-pasted [this code](https://github.com/bhauman/lein-figwheel/blob/master/example/src/example/server.clj) from your example and I wasn't able to make it work. It looks like figwheel just ignores the handler specified in project.clj using :ring-handler

I hope I'll have some time tomorrow to have a look.

Thanks.
